### DISCUSSION TITLE
Can we implement this charts on Vue js ?

### DISCUSSION THREAD

**Discussion by amirsafaricg** [tradingview/lightweight-charts#964] [Q&A]
I have a vue project and i wanted to add my custom data to the chart so i need this library to work on Vue js 3 

is there anyway to do it ?

**Comment by timocov** [comment]
Unfortunately we don't have an example for vuejs at the moment. Do you have any specific issues you're struggling with?

**Reply by amirsafaricg** [reply]
Hi thanks for the reply 
actually not yet I didn't try if it work's or not at all
but as I was reading the discussions it looks like this library is not limited and can be used anywhere (am I right ?)  

I actually don't know for displaying the chart 
what should I write in html 
all I found in doc's and examples was Js syntax

**Reply by amirsafaricg** [reply]
https://ibb.co/jzMk5CK
https://ibb.co/QKkt9km
https://ibb.co/dmXj1Zj

some images showing what im stuck at 
btw im using Quasar a library built with Vue.js 
https://quasar.dev/

**Reply by timocov** [reply]
The issue tells you that the library cannot find an element with id you provided as a container on the page. I'd suggest you to provide a reference to a DOM element instead (which you can get from your framework or create manually and then render it to your page).

**Reply by amirsafaricg** [reply]
I found the fix 
ill post it in the disscusions (solution)

**Comment by amirsafaricg** [comment]
I dropped example codes in my Vue project 
and made an <div> with the Id of container 
and what happened was that the i got the error that the plugin wasnt able to find the element in the DOM and didn't work
